src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/migration/InmateStatusOnSubmissionMigrationJobTest.kt

```diff
@@ -0,0 +1,310 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.unit.migration
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.migration.determineInOutStatus
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.prisonsapi.InOutStatus
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.prisonsapi.PrisonPeriod
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.prisonsapi.PrisonerInPrisonSummary
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.prisonsapi.SignificantMovements
+import java.time.LocalDateTime
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+class InmateStatusOnSubmissionMigrationJobTest {
+
+  @Test
+  fun `Determine In Out Status returns OUT if no Prison Periods provided`() {
+    val result = determineInOutStatus(
```

Comment:
Big fan of unit testing all this stuff here, rather than testing all the possible permutations in the integration tests